vocabularies/Common.xml

```diff
@@ -1454,6 +1454,105 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
       </Term>
 
+      <!--
+      <Term Name="RecommendationsFunction" Type="Common.QualifiedName" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Qualified name of a function that computes AI-based recommendations for entities of the annotated type (see [this template](#GetRecommendations))" />
+      </Term>
+      -->
+      <Function Name="GetRecommendations" IsBound="true">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.IsSignature" />
+        <Annotation Term="Core.Description" String="Computes AI-based recommendations for the bound entity and its related entities" />
+        <Annotation Term="Core.LongDescription">
+          <String>The function SHOULD compute the recommendations asynchronously, see [this diagram](../docs/recommendations.md).</String>
+        </Annotation>
+        <Parameter Name="Entity" Type="Edm.EntityType" Nullable="false" />
+        <Parameter Name="Targets" Type="Common.RecommendationsTargetsType" Nullable="false">
+          <Annotation Term="Core.Description" String="Entities, properties or action/function parameters for which recommendations shall be retrieved" />
+          <Annotation Term="Core.LongDescription">
+            <String>Implementations of this function MAY omit this parameter.
+            If this parameter is omitted (by the implementation or in the invocation), all recommendations are retrieved.</String>
+          </Annotation>
+          <Annotation Term="Core.OptionalParameter">
+            <Record />
+          </Annotation>
+        </Parameter>
+        <ReturnType Type="Collection(Common.PropertyRecommendationType)" Nullable="false">
+          <Annotation Term="Core.Description" String="A collection of recommendations that differ in their `AIRecommendedFieldPath` property" />
+        </ReturnType>
+      </Function>
+      <ComplexType Name="RecommendationsTargetsType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="URLs" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.IsURL" />
+          <Annotation Term="Core.Description" String="URLs (relative to the request URL) that address entities, properties or action/function parameters for which recommendations shall be retrieved" />
+          <Annotation Term="Core.LongDescription">
+            <String>The request `SalesOrder('A')/ns.GetRecommendations?Targets={"URLs":["","Items(10)"]}`
+            retrieves recommendations for the entities `SalesOrder('A')` and `SalesOrder('A')/Items(10)`.</String>
+          </Annotation>
+        </Property>
+      </ComplexType>
+      <ComplexType Name="PropertyRecommendationType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="AIRecommendedFieldPath" Type="Edm.String" Nullable="false">
```

Comment:
```suggestion
        <Property Name="RecommendedFieldPath" Type="Edm.String" Nullable="false">
```
I would keep it implementation-neutral

Reply:
The names are aligned with the Global Field Name governance, they insisted on the AI prefix to distinguish them from existing names (for example, for product recommendations).